src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+    <link rel="stylesheet" href="./style.css">
+    <script src="https://kit.fontawesome.com/9b54de88d2.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="to-do-list">
+        <div class="header">
+            <span>Today's To Do</span>
+            <i class="fa-solid fa-arrows-rotate"></i>
+        </div>
+
+        <div class="add-task-section">
+            <form action="">
+                <div><input class="enter-task" type="text" placeholder="Add to your list.." required></div>
+            </form>
+        </div>
+
+        <div class="tasks-section">
```

Comment:
Instead of a Div Element with a class section, I suggest you use an actual section element as this will be semantically be of importance.